docs/runbooks/NoReadyVirtController.md

```diff
@@ -6,7 +6,7 @@
 
 This alert fires when no available `virt-controller` devices have been detected for 5 minutes.
 
-A `virt-controller` handles monitoring the custom resource definitions (CRDs) of a virtual machine instance (VMI) and managing the associated pods. The device creates pods for VMIs and manages the lifecycle of the pods.
+The virt controller handles monitoring the custom resource definitions (CRDs) of a virtual machine instance (VMI) and managing the associated pods. The device creates pods for VMIs and manages the lifecycle of the pods.
```

Comment:
Hmm. Now it looks weirder without the markup! Sorry to go back and forth on this. :-(

Here's my suggestion:

"The `virt-controller` devices monitor the custom resource definitions of virtual machine instances (VMIs) and manage the associated pods. The devices create pods for VMIs and manage the lifecycle of the pods."

The second sentence might be redundant but I don't think it really matters in a runbook, so we don't need to fuss over it.